docs/en/enterprise-edition/policy-reference/azure-policies/azure-networking-policies/bc-azure-2-49.adoc

```diff
@@ -0,0 +1,68 @@
+
+== Azure Machine Learning workspace is not configured with overly permissive network access
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+| TBD
+
+|Checkov ID
+| https://github.com/bridgecrewio/checkov/blob/main/checkov/terraform/checks/graph_checks/azure/AzureMLWorkspacePublicNetwork.yaml[CKV2_AZURE_49]
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Terraform,TerraformPlan,ARM
+
+|===
+
+=== Description
+
+This policy validates that your Azure Machine Learning workspace is not set up with too broad network access. This is a measure against potential security risks, as having overly permissive network access can lead to unauthorized access, data breaches, and loss of sensitive data. Maintaining stringent network access control ensures only authorized personnel, applications or machines can access or change your data. The policy ensures that only the necessary, trusted entities have access to your machine learning workspace. This strengthens the security stance and helps protect your resources.
```

Comment:
This policy checks whether Azure Machine Learning workspaces have overly permissive network access. Overly permissive access creates security risks by allowing unauthorized entities such as  personnel, applications or machines to access or modify your data. By enforcing strict network access controls, only authorized entities can connect to your workspace, strengthening your security posture and protecting your valuable machine learning resources.